0000-template.md

```diff
@@ -90,3 +85,12 @@ related issue: (tracking issue number)
 >
 > If you have tried and cannot think of any future possibilities, you may simply
 > state that you cannot think of anything.
+
+# Implementation Plan
```

Comment:
Implementation plans change constantly. I'd rather not put those details in the RFC.

How about we change this title to 'Changes Required' or something along those lines, to summarize what components of the system need to change and how (briefly).

This will keep the RFC nice and clean, focusing only on the design and CX elements of the feature.

Reply:
One of the goals of this addition is to introduce a formal step in which we review an implementation plan, and I didn't want to introduce yet another document just for that. This is why the template indicates that this should only be filled out when the RFC is ready to be implemented.

I'll revise this slightly to offer users to reference a GitHub Project Board or reference a separate document that includes the implementation plan in order to allow a bit more flexibility.

Would that work?

Reply:
Slightly reworded. I still think there is value in adding a link to the plan once planning is performed.